### ISSUE TITLE
make user info clickable

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/android#639]
#585 implements the new user info screen. It displays things like email, address, phone, twitter, website.

All these info blocks should be clickable launching the correspnding action, like URL in browser, mail client, twitter/browser, phone-call.

**Comment by mario** [comment]
Also @tobiasKaminsky - please confirm and I'll mark is as approved.

**Comment by mario** [comment]
Maybe not, but this might be used for showing info for other users. As for copy ... maybe long click on the field?

**Comment by tobiasKaminsky** [comment]
You mean like "hey wanna see my twitter account, I just open my nc app and click on my twitter info"? ;-)
I still not really see the use case of it... ;-)

**Comment by mario** [comment]
No, I mean ... eventually I might be able to see your profile inside the app :P

**Comment by tobiasKaminsky** [comment]
If/when this should be possible then we should enhance the user info view in the way @AndyScherzinger proposed, but for now...?

**Comment by AndyScherzinger** [comment]
I am fine with leaving it as is for now and enhance when the corresponding service is available.

**Comment by tobiasKaminsky** [comment]
Why is this approved @eppfel when @AndyScherzinger and me vote to add this only if other profiles are visible?
Or do you refer to "clickable" as "copyable"?

**Comment by eppfel** [comment]
Ah, sorry. I guess I misinterpreted approved then. I thought approved does not have a time/release implication, but just means will be added at some point....

**Comment by tobiasKaminsky** [comment]
@eppfel yes, this is what approved means. 
But do we already have a conclusion? Andy proposed that clicking leads to the function, e.g. call. I proposed that a click leads to copy. Andys last comment was to leave it the way it is.
And you approved it...but what is approved? ;-)

I would add a copy on long-press. This way the user can at least use the text.
And, in a new issue, if it is possible to see other profiles (which is not possible on server-side) then we can enhance it to open the corresponding function, e.g. call.

**Comment by jancborchardt** [comment]
In the future, you will of course also be able to see the profiles of other people on click of their name/avatar. To have a consistency and learnability of functionality here, we should make tap do the action, and long-press copy the content.

Ok?

**Comment by tobiasKaminsky** [comment]
Fine for me 👍

**Comment by SanjaySargam** [comment]
Can I contribute?

**Comment by jld3103** [comment]
Yes!

**Comment by SanjaySargam** [comment]
Plz assign me?

**Comment by jld3103** [comment]
done

**Comment by sam2002jain** [comment]
Hello!! please assign to me I can make it more optimize and clean
